ssg/templates.py

```diff
@@ -1,3 +1,7 @@
+"""
```

Comment:
I think I put the comment to the wrong line, I am sorry. I think you fixed it because I posted a similar comment later pointing to correct line. Disregard this one.